test/SlowTests/Issues/RavenDB-19544.cs

```diff
@@ -16,10 +17,11 @@ public RavenDB_19544(ITestOutputHelper output) : base(output)
     {
     }
 
-    [Fact]
-    public async Task CanGetSpatialDistanceOnIndex_MapReduce()
+    [RavenTheory(RavenTestCategory.Querying | RavenTestCategory.Spatial)]
+    [RavenData(DatabaseMode = RavenDatabaseMode.Single, SearchEngineMode = RavenSearchEngineMode.Lucene)]
```

Comment:
Can you add a comment here?
`// [RavenData(DatabaseMode = RavenDatabaseMode.Sharded, SearchEngineMode = RavenSearchEngineMode.Lucene, Skip = "Distance sorting is not supported for Map-Reduce indexes in sharding as of yet")]`

Reply:
for the future, if someone will want to migrate it to the sharded version